src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,376 @@
+
+#### Image Acquisition
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Key name**               | **Requirement level** | **Data type**            | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| -------------------------- | --------------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| PixelSize                  | REQUIRED              | [array][] of [numbers][] | A 2- or 3-number array of the physical size of a pixel, either`[PixelSizeX, PixelSizeY]` or `[PixelSizeX, PixelSizeY, PixelSizeZ]`, where X is the width, Y the height and Z the depth. If the file format is OME-TIFF, these values need to be consistent with `PhysicalSizeX`, `PhysicalSizeY` and `PhysicalSizeZ` OME metadata fields, after converting in `PixelSizeUnits` according to `PhysicalSizeXunit`, `PhysicalSizeYunit` and `PhysicalSizeZunit` OME fields. |
+| PixelSizeUnits             | REQUIRED              | [string][]               | Unit format of the specified PixelSize. MUST be micrometer (`"um"`).                                                                                                                                                                                                                                                                                                                                                                                                     |
+| Immersion                  | OPTIONAL              | [string][]               | Lens immersion medium. If the file format is OME-TIFF, the value MUST be consistent with the `"Immersion"` OME metadata field.                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Are there reserved keywords here or is it free text?

Reply:
It is intended as free-text.
However, there is an optional field in OME-TIFF metadata that restricts the possible values. So in case this field is present in both JSON metadata and OME metadata, they must match.
For other file formats (ex: PNG) or if the field in not present in OME metadata, there is no restriction.